data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,531 @@
+			`	"Beware, for there are no rules, other than the fact you may only use this one ship. Tell me then, what say you? Will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you."`
+					decline
+			`	"Good! I will of course not be fighting you personally, as it is too risky to do so. I would also not want to deny my brethren the opportunity to prove themselves against a mighty human warrior!"`
+			`	You are not sure if they are being serious or not, but their friends seem to be in for it. Their leader approaches you. "So you think you deserve to fly this mighty ship of the Hai empire?"`
+			`	"Yeah, what makes you think that, human?" Another one adds.`
+			choice
+				`	(Say nothing.)`
+				`	"Are you trying to scare me out of the fight?"`
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. You should see it more as a fight for you to truly earn your place with this ship, but as far as I am concerned, you are an enemy. Make no mistakes about that, this fight has only two rules: nobody other than us here may interfere, and being disabled results in losing the fight. Do not even think about tricking us by switching to something other than this Ladybug, or we will not bother showing up! You must prove your worth for this prize ship in this ship and with only this ship."`
+			`	"And do not even think about capturing our ships. Errors happen, so we can understand some may die in glorious battle, but otherwise looting us is as far as we will allow!"`
+			# This is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (But in reality with disabling a lot of ships it goes faster than that)
+			`	They suddenly look at you intensively, straight in the eyes, "We sometimes bend the rules for our friends, but that is not your case," they say with a sudden change of tone, before leaving.`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	The engineer comes back, looking a bit impatient, "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships to get the initiative. The engineer utters a word to you once they are far enough away, "I want this done by the end of the day, do not try my patience. You are lucky I offered this deal in the first place, make sure I do not regret it. Now, quick, get to your ship!" You see the contestants have already begun the launching procedures and launch as soon as you can.`
```

Comment:
```suggestion
			`	The engineer comes back, looking a bit impatient. "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships. The engineer pulls you in close once they are far enough away. "I want this done by the end of the day, so do not try my patience. You are lucky I offered this deal in the first place. Now, quick, get to your ship!" You see the contestants have already begun the launching procedures and launch as soon as you can.`
```
Why are we lucky the engineer offered this "deal"? As far as we know, we bought a ship and the engineer just said "you have to fight for this". Doesn't exactly track. How is what the engineer offered better than other outcomes? You need to explain why we're "lucky" more or just remove the mention.

Reply:
the others would have just killed you and taken the ship but she had arrangements with the sympathizers